karate-tests/src/test/java/patients/healthcareWorker/patchPatient.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Patch patient - Healthcare worker access mode
+
+  Background:
+    * def utils = call read('classpath:helpers/utils.feature')
+    * def accessToken = karate.callSingle('classpath:patients/healthcareWorker/auth-redirect.feature').accessToken
+    * def requestHeaders = call read('classpath:patients/healthcareWorker/healthcare-worker-headers.js')
+    * configure headers = requestHeaders 
+    * url baseURL
+    * def nhsNumber = karate.get('nhsNumber', '9912002725')
+    * path 'Patient', nhsNumber
+    * method get
+    * status 200
+    
+    * def patientObject = response
+    * def etag = karate.response.header('etag')
+    * def originalVersion = parseInt(response.meta.versionId)
+
+  @gender
+  Scenario: Change the gender of a patient
+    * def currentGender = patientObject.gender
+    * def pickNewGender = 
+      """
+        function(currentGender) {
+          const genderOptions = ['male', 'female', 'unknown'];
+          var newGender = 'female'; 
+          while (currentGender == newGender) {
+            newGender = genderOptions[Math.floor(Math.random() * genderOptions.length)];
```

Comment:
This doesn't seem efficient. We can remove the currentGender value from genderOptions and pick at random an option from the ones left.

Reply:
That's a good idea. I might also move this into the collection of utility functions.

Reply:
Okay I made a new function in utils.feature for this:

```js
function(optionsArray, itemToIgnore) {
      const index = optionsArray.indexOf(itemToIgnore);
      optionsArray.splice(index, 1)
      return optionsArray[Math.floor(Math.random() * optionsArray.length)]
}
```